community/releases/versions.md

```diff
@@ -5,6 +5,22 @@ sidebar:
   nav: community
 ---
 
+The Spinnaker releases listed below are top-level versions tying together each
+Spinnaker subcomponents' versions. These have been validated together in an 
+end-to-end integration test suite curated by the Spinnaker community, and
+represent a more maintainable, easy to upgrade Spinnaker. While 
+it's still possible to install each Spinnaker subcomponent's versions 
```

Comment:
There is not - and I've only seen it asked about once by a user with very unique constraints. It's kind of an advanced topic and this doesn't feel like the right place to bring it up.